client/web/compose/src/components/PageBlocks/Wrap/base.vue

```diff
@@ -18,6 +18,12 @@ export default {
     },
   },
 
+  data () {
+    return {
+      unsavedChange: false,
```

Comment:
Please no, you can pass a prop to it that will determine this. And you evaluate it on the parent (builder)